release.json

```diff
@@ -44,6 +44,30 @@
         "WINDOWS_APMINJECT_VERSION": "1.1.1",
         "WINDOWS_APMINJECT_SHASUM": "c5c228f6030ce2e19b8bfb28bd054dc246c7f1c799431e73018e6cb46ee59d2f"
     },
+    "dev": {
```

Comment:
Why not name it `dev-a6`? Like we do for `release-a6` and `release-a7`

Reply:
Do we really need separate entries for `dev` and `nightly`? In which case would these be different?

Reply:
🤔 we don't need separate entry, I just thought it would be less intuitive to ask to edit nightly when making `dev` builds. But I can't think of any scenario where they should be different so I might remove it all together to avoid repetition.